boards/arm/bt610/bt610.dts

```diff
@@ -39,12 +39,12 @@
 		compatible = "pwm-leds";
 
 		led1pwm: led_1_pwm {
-			pwms = <&pwm0 39>;
+			pwms = <&pwm0 0 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
 			label = "Red PWM LED 1";
 		};
 
 		led2pwm: led_2_pwm {
-			pwms = <&pwm1 35>;
+			pwms = <&pwm1 0 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
```

Comment:
Should be high to be on, seems the polarity here is wrong?

Reply:
Right, my mistake. Thanks for catching this.

Reply:
Fixed.